src/hist_smooth/smoothing_api.py

```diff
@@ -0,0 +1,82 @@
+from hist_smooth.base_smoothing import (
+    moving_median_odd,
+    hanning,
+    quadratic_interpolation,
+    twicing,
+)
+
+from hist_smooth.histogram_smoothing import (
+    smooth_histogram
+)
+
+from hist_smooth.tukey_smoothing import (

```

Comment:
what will be the use of the pre-defined algorithms in this module? will they be somehow exposed to the user?